wp-parsely.php

```diff
@@ -83,6 +83,7 @@ public function __construct() {
         // inserting parsely code
         add_action('wp_head', array($this, 'insert_parsely_page'));
         add_action('wp_footer', array($this, 'insert_parsely_javascript'));
+        add_action('instant_articles_compat_registry_analytics', array($this, 'insert_parsely_tracking_fbia'));
```

Comment:
Scratch that! I didn't initially understand the mechanism there for hooking into `instant_articles_compat_registry_analytics`